enterprise/models.py

```diff
@@ -300,6 +302,9 @@ class PendingEnterpriseCustomerUser(TimeStampedModel):
     enterprise_customer = models.ForeignKey(EnterpriseCustomer, blank=False, null=False)
     user_email = models.EmailField(null=False, blank=False, unique=True)
 
+    class Meta(object):
+        app_label = 'enterprise'
```

Comment:
I have been getting warnings when starting up the LMS in devstack:
RemovedInDjango19Warning: Model class enterprise.models.EnterpriseCustomerIdentityProvider doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.